### PR TITLE
Qt: Option to disable keyboard hotkeys

### PR DESCRIPTION
**Pull request by DerTodIstEinDandy**

### CODE REVIEW

File: rpcs3/rpcs3qt/tooltips.h [hunk 1]
```diff
@@ -103,6 +103,7 @@ class Tooltips : public QObject
 		const QString resize_on_boot               = tr("Automatically resizes the game window on boot.\nThis does not change the internal game resolution.");
 		const QString show_trophy_popups           = tr("Show trophy pop-ups when a trophy is unlocked.");
 		const QString disable_mouse                = tr("Disables the activation of fullscreen mode per double-click while the game screen is active.\nCheck this if you want to play with mouse and keyboard (for example with UCR).");
+		const QString disable_kb_hotkeys           = tr("Disables keyboard hotkeys such as Ctrl-S, Ctrl-E, Ctrl-R, Ctrl-P while the game screen is active. \nCheck this if you want to play with mouse and keyboard");
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/tooltips.h:106]:
```suggestion
		const QString disable_kb_hotkeys           = tr("Disables keyboard hotkeys such as Ctrl-S, Ctrl-E, Ctrl-R, Ctrl-P while the game screen is active.\nCheck this if you want to play with mouse and keyboard");
```

File: rpcs3/rpcs3qt/tooltips.h [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#pragma once
+﻿#pragma once
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/tooltips.h:1]: remove this change please.

File: rpcs3/rpcs3qt/settings_dialog.ui [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.ui:1]: remove this change please

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 4]
```diff
@@ -1265,14 +1265,23 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	{
 		SubscribeTooltip(ui->gs_resizeOnBoot, tooltips.settings.resize_on_boot);
 		SubscribeTooltip(ui->gs_disableMouse, tooltips.settings.disable_mouse);
+		SubscribeTooltip(ui->gs_disableKbHotkeys, tooltips.settings.disable_kb_hotkeys);
 		SubscribeTooltip(ui->gs_showMouseInFullscreen, tooltips.settings.show_mouse_in_fullscreen);
 
-		ui->gs_disableMouse->setChecked(m_gui_settings->GetValue(gui::gs_disableMouse).toBool());
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.cpp:1270]: these changes are unnecessary

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 5]
```diff
@@ -1265,14 +1265,23 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
...
+		const bool disable_mouse = m_gui_settings->GetValue(gui::gs_disableMouse).toBool();
+		ui->gs_disableMouse->setChecked(disable_mouse);
+		connect(ui->gs_disableMouse, &QCheckBox::clicked, [=, this](bool val)
 		{
 			m_gui_settings->SetValue(gui::gs_disableMouse, val);
 		});
 
+		const bool disable_kb_hotkeys = m_gui_settings->GetValue(gui::gs_disableKbHotkeys).toBool();
+		ui->gs_disableKbHotkeys->setChecked(disable_kb_hotkeys);
+		connect(ui->gs_disableKbHotkeys, &QCheckBox::clicked, [=, this](bool val)
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.cpp:1280]:
```suggestion
		connect(ui->gs_disableKbHotkeys, &QCheckBox::clicked, [this](bool val)
```

File: rpcs3/rpcs3qt/tooltips.h (hunk 2 above)
Comment (DerTodIstEinDandy) [reply rpcs3/rpcs3qt/tooltips.h]: seems to be fixed now

**Pull request merged**